ffcx/codegeneration/symbols.py

```diff
@@ -92,6 +92,17 @@ def argument_loop_index(self, iarg):
         indices = ["i", "j", "k", "l"]
         return self.S(indices[iarg])
 
+    def get_reflection(self, L, i):
```

Comment:
Just `reflection` here, other FFCXBackendSymbols doesnt have `get_*` format.

Reply:
I've renamed it `entity_reflection`, as `reflection` could mean a lot of things